src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final String defaultColor = ColorSupplier.Color.WHITE.toString();
+    private final int defaultRadius = 10;
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        Random random = new Random();
+        Shape shape = Shape.values()[random.nextInt(Shape.values().length)];
+        String color = new ColorSupplier().getRandomColor();
+        int firstSize = random.nextInt(Figure.MAX_SIDE);
+        switch (shape) {
+            case CIRCLE:
+                figure = new Circle(color, firstSize);
+                break;
```

Comment:
do the same for each case
```suggestion
                return new Circle(color, firstSize);
```